src/integrationTest/java/uk/gov/hmcts/reform/managecase/fixtures/WiremockFixtures.java

```diff
@@ -66,27 +70,28 @@ public String getName() {
         public ResponseDefinition transform(Request request, ResponseDefinition responseDefinition,
                                             FileSource files, Parameters parameters) {
             return ResponseDefinitionBuilder.like(responseDefinition)
-                    .withHeader(HttpHeaders.CONNECTION, "close")
-                    .build();
+                .withHeader(HttpHeaders.CONNECTION, "close")
+                .build();
         }
     }
 
     public static void stubGetUsersByOrganisation(FindUsersByOrganisationResponse response) {
         stubFor(WireMock.get(urlEqualTo("/refdata/external/v1/organisations/users?status=Active&returnRoles=false"))
-                .willReturn(okForJson(response)));
+                    .willReturn(okForJson(response)));
     }
 
     public static void stubSearchCaseWithPrefix(String caseTypeId, String searchQuery,
                                                 CaseDetails caseDetails, String prefix) {
         stubFor(WireMock.post(urlEqualTo(prefix + SEARCH_CASES + "?ctid=" + caseTypeId))
-            .withRequestBody(equalToJson(searchQuery))
-            .withHeader(AUTHORIZATION, equalTo(SYS_USER_TOKEN))
-            .withHeader(SERVICE_AUTHORIZATION, equalTo(S2S_TOKEN))
-            .willReturn(aResponse()
-                .withStatus(HTTP_OK)
-                    .withBody(getJsonString(
-                        caseDetails == null ? new CaseSearchResponse() : new CaseSearchResponse(list(caseDetails))))
-                .withHeader("Content-Type", "application/json")));
+                    .withRequestBody(equalToJson(searchQuery))
+                    .withHeader(AUTHORIZATION, equalTo(SYS_USER_TOKEN))
+                    .withHeader(SERVICE_AUTHORIZATION, equalTo(S2S_TOKEN))
+                    .willReturn(aResponse()
+                                    .withStatus(HTTP_OK)
+                                    .withBody(getJsonString(
+                                        caseDetails == null ? new CaseSearchResponse() : new CaseSearchResponse(list(
+                                            caseDetails))))
+                                    .withHeader("Content-Type", "application/json")));
```

Comment:
`.withHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)));`



as it is used elsewhere in this class?

Reply:
done